### PR TITLE
fix(requirements): prop up python-driver

### PR DESCRIPTION
**Pull request by dkropachev**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by fruch** [comment]
@dkropachev can you solve the conflict and get this merged ?

File: docker/env/version [hunk 1]
```diff
@@ -1 +1 @@
-0.99
+1.00
```
[the remark this answers is not in the document]
Comment (bentsi) [reply docker/env/version:1]: should be 1.02

[the remark this answers is not in the document]
Comment (dkropachev) [reply docker/env/version]: done

**Review by bentsi — DISMISSED** [review]

**Comment by bentsi** [comment]
please check why it can't find Argus, probably you built the image using old Python requirements file

**Review by fruch — APPROVED** [review]
LGTM

**Comment by dkropachev** [comment]
> please check why it can't find Argus, probably you built the image using old Python requirements file

Argus issue has been fixed

**Pull request closed without merging**